ui/src/tasks/actions/index.ts

```diff
@@ -294,7 +301,10 @@ export const removeTaskLabelAsync = (taskID: string, label: Label) => async (
   dispatch
 ): Promise<void> => {
   try {
-    await apiDeleteTasksLabel({taskID, labelID: label.id})
+    const deleteResp = await apiDeleteTasksLabel({taskID, labelID: label.id})
```

Comment:
nit - sometimes the response is called `resp` and sometimes it is called `postResp`- I would prefer uniformity and  `resp`

Reply:
@ebb-tide since we're already declaring a `resp` further down, I had to assign a different name:

https://github.com/influxdata/influxdb/pull/16400/files/55bd264cd5d7fb5b9e2243b21cf3adfc9d751510#diff-3084610c031fd81910381dcc8507c9fcR308

Reply:
ah ok.

Reply:
@Asalem1 @ebb-tide is this an indicator that this operation should be in it's own try/catch block? i think it should since these seem to be discrete actions - `apiGetTask`  doesn't seem to use the response from `apiDeleteTasksLabel`, so they really don't need to be grouped together. the main reason I can think of is to prevent duplication of the `catch` block, but I think duplication is less detrimental than muddled intent.

if they must stay together, i think `resp` should be changed to `getTaskResp` to be in line with `deleteResp`. calling one `deleteResp` and one `resp` gives a bit more weight to `resp` and I'm not sure if that's the intent here.